src/services/poi.ts

```diff
@@ -29,3 +34,34 @@ export async function fixGeometryOfPoi(rennesApp: RennesApp) {
     )
   })
 }
+
+export function displayNameOfSelectedPoi(feature: Feature<Geometry>) {
+  const map3dStore = useMap3dStore()
+  if (feature === null) return
+  const styleItem = generatePoiStyle(
+    feature.getProperties()['site_nom'],
+    feature.getProperties()['distance'],
+    map3dStore.is3D(),
+    false
+  )
+  feature.setStyle(styleItem.style)
+}
+
+export function undisplayPoiExpectCurrent() {
```

Comment:
Could you add some comments about this function? The name is getting quite "strange" :)